tests/test_vms_exist.py

```diff
@@ -176,6 +181,11 @@ def sd_log_template(self):
         self.assertTrue('sd-workstation' in vm.tags)
         self.assertFalse(vm.template_for_dispvms)
         self._check_kernel(vm)
+        # Check the size of the private volume
+        # Should be 5GB
+        # >>> 1024 * 1024 * 5 * 1024
+        vol = vm.volumes["private"]
+        self.assertEqual(vol.size, 5368709120)
```

Comment:
Here, you are testing the template vm, not the AppVM (sd-log). Strangely though, this test is not failing. Perhaps it is not running?